qa/runtests.sh

```diff
@@ -62,13 +62,8 @@ function setup_virtualenv {
     printf "%sPython virtual environment initialized%s\n" "${YELLOW}" "${NORMAL}"
 
     pip install -U pip
-    pip3.6 install -r requirements.txt
-
-    # Install profiling module
-    pip3.6 install pytest-profiling
-
-    # Install pytest-coverage module
-    pip3.6 install pytest-cov
+    pip install -r requirements.txt
+    pip install -r test/requirements.txt
```

Comment:
Do a atomic install.
```suggestion
    pip install -r requirements.txt -r test/requirements.txt
```